spacelift/data_stack.go

```diff
@@ -67,6 +80,11 @@ func dataStack() *schema.Resource {
 				Description: "Terraform version to use",
 				Computed:    true,
 			},
+			"vcs_provider": &schema.Schema{
```

Comment:
You don't need this field now. If the `gitlab` list is empty, it implies that VCS provider is GitHub.